dot/parachain/overseer/overseer.go

```diff
@@ -29,7 +29,8 @@ import (
 )
 
 var (
-	logger = log.NewFromGlobal(log.AddContext("pkg", "parachain-overseer"))
+	logger               = log.NewFromGlobal(log.AddContext("pkg", "parachain-overseer"))
+	ParachainHostAPIName = common.MustBlake2b8([]byte("ParachainHost"))
```

Comment:
The runtime `version` struct has an array field called `apis`, each item in the array is a tuple of `Name [8]byte` and `Version uint32`, where the name field is the `blake2b` hash of the api's name.